src/core/player.js

```diff
@@ -41,15 +41,27 @@ DM.provide('',
     destroy: function(id)
     {
         if (!id) {  // destroy all players of the page
-            if (DM.Array.keys(DM.Player._INSTANCES).length === 0)
-                throw new Error("DM.destroy(): no player to destroy");
+            if (DM.Array.keys(DM.Player._INSTANCES).length === 0) {
+                try {
```

Comment:
@Supergallus What about factorizing this `try/catch` block? It is repeated several time over the file, and there are already `DM.log()` and `DM.error()` available there: https://github.com/dailymotion/dailymotion-sdk-js/blob/8eff611c8cc42d2dcd7d54e17917021769689bc0/src/core/prelude.js#L183-L226